test/integration/short/client-execute-tests.js

```diff
@@ -539,6 +539,55 @@ describe('Client', function () {
         done();
       });
     });
+    vdescribe('3.0.16', 'with noCompact', function () {
+      before(function (done) {
+        // C* 4.0 does not support compact storage
+        if (helper.isCassandraGreaterThan('4.0')) {
+          this.skip();
+        }
+        const client = newInstance({keyspace: keyspace, protocolOptions: { noCompact: true }});
+        utils.series([
+          client.connect.bind(client),
+          helper.toTask(client.execute, client, 'CREATE TABLE tbl_cs (key blob PRIMARY KEY, bar int, baz uuid) WITH COMPACT STORAGE'),
+          helper.toTask(client.execute, client, "INSERT INTO tbl_cs (key, bar, baz, column1, value) values (0xc0, 10, 33cb65d4-6721-4ca8-854f-1f020c5353cb, 'yak', 0xcafedead)"),
+        ], done);
+      });
+      it('set to true should reveal non-schema columns', () => {
```

Comment:
Its awesome that you found a way to cover it using integration tests (I will use the same for C# driver as well, <insert_burns_meme_here> mwahahaha).